diff --git pkg/instance/start.go pkg/instance/start.go
index 8b3f15b3..fd2b2488 100644
--- pkg/instance/start.go
+++ pkg/instance/start.go
@@ -332,7 +332,7 @@ func Start(ctx context.Context, inst *limatype.Instance, launchHostAgentForegrou
 
 func waitHostAgentStart(_ context.Context, haPIDPath, haStderrPath string) error {
 	begin := time.Now()
-	deadlineDuration := 5 * time.Second
+	deadlineDuration := 30 * time.Second
 	deadline := begin.Add(deadlineDuration)
 	for {
 		if _, err := os.Stat(haPIDPath); !errors.Is(err, os.ErrNotExist) {
@@ -341,6 +341,7 @@ func waitHostAgentStart(_ context.Context, haPIDPath, haStderrPath string) error
 		if time.Now().After(deadline) {
 			return fmt.Errorf("hostagent (%#q) did not start up in %v (hint: see %#q)", haPIDPath, deadlineDuration, haStderrPath)
 		}
+		time.Sleep(100 * time.Millisecond)
 	}
 }
 
