--- src/CarbonEvent.zig.orig
+++ src/CarbonEvent.zig
@@ -1,4 +1,4 @@
 const std = @import("std");
-const c = @import("c.zig");
+const c = @import("c.zig").c_impl;
 
 const CarbonEvent = @This();

--- src/echo.zig.orig
+++ src/echo.zig
@@ -3,5 +3,5 @@
 const Keycodes = @import("Keycodes.zig");
 
-const c = @import("c.zig");
+const c = @import("c.zig").c_impl;
 
 extern fn NSApplicationLoad() void;

--- src/EventTap.zig.orig
+++ src/EventTap.zig
@@ -1,4 +1,4 @@
 const std = @import("std");
-const c = @import("c.zig");
+const c = @import("c.zig").c_impl;
 
 handle: c.CFMachPortRef = null,

--- src/exec.zig.orig
+++ src/exec.zig
@@ -1,3 +1,3 @@
-const c = @import("c.zig");
+const c = @import("c.zig").c_impl;
 const std = @import("std");
 

--- src/Hotload.zig.orig
+++ src/Hotload.zig
@@ -1,4 +1,4 @@
 const std = @import("std");
-const c = @import("c.zig");
+const c = @import("c.zig").c_impl;
 
 /// File system event monitoring using macOS FSEvents API.

--- src/Keycodes.zig.orig
+++ src/Keycodes.zig
@@ -1,4 +1,4 @@
 const std = @import("std");
-const c = @import("c.zig");
+const c = @import("c.zig").c_impl;
 const log = std.log.scoped(.keycodes);
 

--- src/Parser.zig.orig
+++ src/Parser.zig
@@ -1,4 +1,4 @@
 const std = @import("std");
-const c = @import("c.zig");
+const c = @import("c.zig").c_impl;
 const Tokenizer = @import("Tokenizer.zig");

--- src/service.zig.orig
+++ src/service.zig
@@ -1,4 +1,4 @@
 const std = @import("std");
 const builtin = @import("builtin");
-const c = @import("c.zig");
+const c = @import("c.zig").c_impl;
 const log = std.log.scoped(.service);

--- src/skhd.zig.orig
+++ src/skhd.zig
@@ -2,5 +2,5 @@
 const builtin = @import("builtin");
 
-const c = @import("c.zig");
+const c = @import("c.zig").c_impl;
 const CarbonEvent = @import("CarbonEvent.zig");

--- src/synthesize.zig.orig
+++ src/synthesize.zig
@@ -1,4 +1,4 @@
 const std = @import("std");
-const c = @import("c.zig");
+const c = @import("c.zig").c_impl;
 const Parser = @import("Parser.zig");
