--- fuse/commands.c.orig
+++ fuse/commands.c
@@ -30,7 +30,7 @@
 #include <getopt.h>
 #include <signal.h>
 #include <sys/socket.h>
-#include <fuse.h>
+#include <fuse3/fuse.h>
 
 #include "lib/afp_internal.h"
 #include "lib/client.h"
--- fuse/daemon.c.orig
+++ fuse/daemon.c
@@ -27,7 +27,7 @@
 #include <time.h>
 #include <sys/time.h>
 #include <unistd.h>
-#include <fuse.h>
+#include <fuse3/fuse.h>
 #include <glob.h>
 
 #include "lib/afp_internal.h"
--- fuse/fuse_fs.c.orig
+++ fuse/fuse_fs.c
@@ -14,7 +14,7 @@
 
 #define HAVE_ARCH_STRUCT_FLOCK
 
-#include <fuse.h>
+#include <fuse3/fuse.h>
 #include <stdio.h>
 #include <string.h>
 #include <errno.h>
--- meson.build.orig
+++ meson.build
@@ -257,7 +257,7 @@
     fuse_readdir_flags_check = cc.compiles(
         '''
         #define FUSE_USE_VERSION 35
-        #include <fuse.h>
+        #include <fuse3/fuse.h>
         enum fuse_readdir_flags flags;
     ''',
         dependencies: fuse_dep,
@@ -269,7 +269,7 @@
     fuse_file_info_ops_check = cc.compiles(
         '''
         #define FUSE_USE_VERSION 35
-        #include <fuse.h>
+        #include <fuse3/fuse.h>
 
         int chown_func(const char *path, uid_t uid, gid_t gid,
                        struct fuse_file_info *fi) { return 0; }
@@ -286,7 +286,7 @@
     fuse_rename_flags_check = cc.compiles(
         '''
         #define FUSE_USE_VERSION 35
-        #include <fuse.h>
+        #include <fuse3/fuse.h>
         int rename_func(const char *from, const char *to,
                         unsigned int flags) { return 0; }
     ''',
@@ -298,7 +298,7 @@
     fuse_utimens_fi_check = cc.compiles(
         '''
         #define FUSE_USE_VERSION 35
-        #include <fuse.h>
+        #include <fuse3/fuse.h>
         int utimens_func(const char *path, const struct timespec tv[2],
                          struct fuse_file_info *fi) { return 0; }
     ''',
